core/src/main/scalanative/sttp/client4/curl/internal/CCurl.scala

```diff
@@ -25,23 +25,29 @@ private[curl] object libcurlPlatformCompat {
 
 @extern
 private[curl] trait CCurl {
+  @name("sttp_curl_setopt_int")
+  def setoptInt(handle: Ptr[Curl], option: CInt, parameter: Int): CInt = extern
+
+  @name("sttp_curl_setopt_long")
+  def setoptLong(handle: Ptr[Curl], option: CInt, parameter: Long): CInt = extern
+
+  @name("sttp_curl_setopt_pointer")
+  def setoptPtr(handle: Ptr[Curl], option: CInt, parameter: Ptr[_]): CInt = extern
```

Comment:
This 3 functions can have regular `setopt` name, `@name` annotations works as Scala Native specific `@targetName` annotation for extern symbols.